### ISSUE TITLE
Cannot insert markdown in the metadata section

### ISSUE THREAD

**Issue by msprotz** [CatalaLang/catala#147]
It seems that I cannot have markdown in-between two "```catala" blocks, in-between `> Begin metadata` and `> End metadata`.

See attached file:
[test.txt](https://github.com/CatalaLang/catala/files/7247465/test.txt)

Thanks,

Jonathan

**Comment by EmileRolley** [comment]
Wouldn't it be more understandable to use something like this?

~~~markdown
## Test

```catala-metadata
declaration scope Foo:
  context bar content date
```

Some text of law that introduces a new concept.

```catala-metadata
declaration enumeration Concept:
  -- Case1
  -- Case2
```
~~~

**Comment by denismerigoux** [comment]
Thanks for chiming in @EmileRolley :) The problem with your suggestion is that it would complicate syntax highlighting since code mode would have two entry markups.

**Comment by denismerigoux** [comment]
Yeah, we could do that without too much of a hassle I guess. @protz  what do you think ?

**Comment by msprotz** [comment]
👍 for Emile's suggestion! thanks!

**Comment by EmileRolley** [comment]
Closed by #150
